marconi-core/src/Marconi/Core/Experiment/Transformer/WithResume.hs

```diff
@@ -139,11 +146,12 @@ class OrdPoint point where
 
 -- | clean the last point, usually because the volatile part of an indexer is no longer valid
 wipeHistory :: ResumeState event -> ResumeState event
-wipeHistory ResumeState{} = ResumeState True Nothing []
+-- wipeHistory ResumeState{} = ResumeState True Nothing []
+wipeHistory (ResumeState _ last' _) = ResumeState False last' []
```

Comment:
We need a session to chat about it: I think that there's a logic problem somewhere else in my implementation that may require this change. Thus, your fix is correct but I'm confused by the logic behind it.

Reply:
Added a TODO for that